Externals/cximage/ximaenc.cpp

```diff
@@ -996,7 +996,7 @@ bool CxImage::CheckFormat(CxFile * hFile, DWORD imagetype)
 ////////////////////////////////////////////////////////////////////////////////
 bool CxImage::CheckFormat(BYTE * buffer, DWORD size, DWORD imagetype)
 {
-	if (buffer==NULL || size==NULL){
+	if (buffer==NULL || size==0){
```

Comment:
`size` is not a pointer.

And changing `buffer==NULL` to `buffer==nullptr` is unneeded, this is a third-party library.

Reply:
Everything is correct here, because `size` is not a pointer.

And changing `buffer==NULL` to `buffer==nullptr` is unneeded, this is a third-party library.